examples/integration/SanitizerApp/BUILD

```diff
@@ -0,0 +1,16 @@
+load("@build_bazel_rules_apple//apple:versioning.bzl", "apple_bundle_version")
```

Comment:
I think instead of having this be another app in the `integrations` example, that we should have a `examples/sanitizers` example, with an app in it for each type of sanitizer. That way we will be able to test the Custom Scheme generation of the various sanitizer checkboxes as well.

Reply:
Thanks, @brentleyjones! Makes sense. I have updated the PR where all the sanitizers are hosted in `examples/sanitizers` with their dedicated app. (currently just the TSan)

> That way we will be able to test the Custom Scheme generation of the various sanitizer checkboxes as well.

I would like your help in understanding this. Do you mean we should have a custom scheme where the .bzl should explicitly say: "Enable Thread Sanitizer" and the project generator should honor this? If this is true, then this would be a part of Run Action (diagnostics as Xcode calls it).

Reply:
Yes, I believe we should (in a future PR) add a `diagnostics` argument (that would take a `xcode_schemes.disagnostics()` function) to the `launch_action()` function, to allow specifying `sanitizers` (and later more of this screen):
<img width="428" alt="CleanShot 2022-09-23 at 16 04 06@2x" src="https://user-images.githubusercontent.com/158658/192056585-6fc089aa-fd37-4a63-a8e8-d6e71d5fb6a9.png">

That way one can have these always enabled for their custom schemes, or generate a scheme for each of the sanitizers.

Reply:
I agree - that would be a better experience!

Reply:
Since this would be taken in a future PR, resolving this for now!